### DISCUSSION TITLE
Version 4.2 Not in Google Play Store?

### DISCUSSION THREAD

**Discussion by jnic40** [osmandapp/OsmAnd#14542] [Q&A] [answered]
This article seems to say that version 4.2 is available in the Android Google Play Store, [https://osmand.net/blog/osmand-android-4-2-released](https://osmand.net/blog/osmand-android-4-2-released). But, I am only seeing version 4.1.11 there. Am I missing something here? There is a specific UTM coordinate display fix that I am wanting in it.
Thanks.

[https://play.google.com/store/apps/details?id=net.osmand](https://play.google.com/store/apps/details?id=net.osmand)

![image](https://user-images.githubusercontent.com/10342403/173590493-e618105e-f7d0-4555-b658-23d3f3e720dc.png)

**Comment by vshcherb** [comment, marked as answer]
Not yet, tomorrow will be a release that probably reach 100% audience. Beta is already available
